src/config_builder.py

```diff
@@ -143,6 +155,11 @@ def _storage_config(self) -> dict:
                 "shared_store": "filesystem",
                 "cache_location": BOLTDB_CACHE_DIR,
             },
+            "tsdb_shipper": {
+                "active_index_directory": TSDB_DIR,
+                "shared_store": "filesystem",
+                "cache_location": TSDB_CACHE_DIR,
+            },
```

Comment:
Should we add a dedicated integration test?
For example:
- Name: `test_boltdb_v11_to_tsdb_v12_migration.py`
- Have a literal bundle that deploys the current loki by revision number (not channel), then upgrade to the current loki.
- Then upgrade again to the current loki.
- Then upgrade again, but override the resource to an old oci-image from before v12 (if this is a use case we want to address).